python/lesson3/tutorial.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Discovering Flask via Python and pip
+---
+
+An alternative to *Ruby's Sinatra* is the lesser known *Flask* (run using Python).
+Both are web application frameworks. Similar to the MVC (model-view-controller)
```

Comment:
Valid point! Appreciated. How the phrase, "An alternative to". 
Does that sound clearer to others. Appreciate people from all different of points of view (regional especially here).